data/styles/clear/include/Basemap.mapcss

```diff
@@ -578,45 +574,102 @@ area|z15-[amenity=parking]
 
 /* 8.BUILDINGS */
 
-/* 8.1 Pier */
+/* 8.1 Pier, dam, cutline */
+
 area|z12-[man_made=pier],
 area|z12-[man_made=breakwater],
 {fill-position: background; fill-opacity: 1; fill-color: @background;casing-color: @background;}
 
 line|z14-[man_made=pier],
 line|z14-[man_made=breakwater],
-line|z14-[man_made=cutline]
-{width: 2;color: @background;opacity: 1;}
+{width: 1.5;color: @background;opacity: 1;}
+
+line|z14-[waterway=dam],
+line|z14-[waterway=weir],
+{width: 1.5;color: @bridge_casing;opacity: 1;}
+
+line|z15[man_made=pier],
+line|z15[man_made=breakwater],
+line|z15[man_made=cutline],
+line|z15[waterway=dam],
+line|z15[waterway=weir],
+{width: 2;}
+
+line|z16[man_made=pier],
+line|z16[man_made=breakwater],
+line|z16[man_made=cutline],
+line|z16[waterway=dam],
+line|z16[waterway=weir],
+{width: 3;}
+
+line|z17[man_made=pier],
+line|z17[man_made=breakwater],
+line|z17[man_made=cutline],
+line|z17[waterway=dam],
+line|z17[waterway=weir],
+{width: 4.5;}
+
+line|z18[man_made=pier],
+line|z18[man_made=breakwater],
+line|z18[man_made=cutline],
+line|z18[waterway=dam],
+line|z18[waterway=weir],
+{width: 6;}
+
+line|z19-[man_made=pier],
+line|z19-[man_made=breakwater],
+line|z19-[man_made=cutline],
+line|z19-[waterway=dam],
+line|z19-[waterway=weir],
+{width: 8;}
+
+line|z14-[man_made=cutline],
+{width: 1;color: @green0;opacity: 1;}
+line|z15[man_made=cutline],
+{width: 1.5;}
+line|z16[man_made=cutline],
+{width: 2;}
+line|z17[man_made=cutline],
+{width: 3;}
+line|z18-[man_made=cutline],
+{width: 4.5;color: @green1;}
```

Comment:
Is it ok green0 before z18 and green1 from z18?

Reply:
its common for vegetation objects to become darker with more zoom;
however I changed it to @green0; only but 0.6 opacity